conf.py

```diff
@@ -0,0 +1,79 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'NumPy Tutorials'
+copyright = '2020, NumPy Community'
+author = 'NumPy Community'
+
+# The full version, including alpha/beta/rc tags
+# release = '0.0.1-dev'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "myst_nb",
```

Comment:
What was the tradeoff between `myst_nb` and `nbsphinx`? I've other projects which use the latter, but if the former was found to be the better choice then I might be interested in swapping over my projects too.

Reply:
I'm not an expert here but I can at least try to provide a little background. `myst-nb` is one part of the larger [executablebooks project](https://github.com/executablebooks) that aims to improve the tooling around the generation of scientific content in a way that supports many input formats. I have not used `nbsphinx` extensively, but the two main reasons I really like `myst-nb` are:
 1. Support for a [text-based notebook format](https://myst-nb.readthedocs.io/en/latest/use/markdown.html)
 2. Built-in support for [caching notebook execution](https://myst-nb.readthedocs.io/en/latest/use/execute.html)

The first point probably isn't so relevant if you already have a bunch of content in traditional `.ipynb` form (though [jupytext supports bi-directional conversion between `ipynb` and `mystnb` format](https://jupytext.readthedocs.io/en/latest/formats.html#myst-markdown), making it easy to try out the text-based format with your existing content).

Execution caching is another nice feature, especially if you have a lot of different source notebooks/files. Note that [jupyter-cache](https://github.com/executablebooks/jupyter-cache) is a separate project, so it's probably possible to get it working with `nbsphinx`. However, it's included by default with `myst-nb` (since it's also part of executablebooks).

There are other reasons as well (`nbsphinx` has an external dependency on `pandoc` whereas (AFAICT) `myst-nb` has no external [non-Python] dependencies). This is by no means an exhaustive list of the differences, just some of the reasons that I proposed `myst-nb` for this project.

Reply:
On the rendering side, I've also noticed that `myst-nb` is a bit more polished, whereas there were some formatting issues with `nbsphinx` (these are cosmetic details but I think those are important here).